src/osd/OSD.cc

```diff
@@ -2314,6 +2314,16 @@ int OSD::get_num_op_threads()
     return get_num_op_shards() * cct->_conf->osd_op_num_threads_per_shard_ssd;
 }
 
+float OSD::get_osd_recovery_sleep()
```

Comment:
@tchaikov yes, I am working further on this. Will it be fine if I change it in the next PR?